subprojects/packagefiles/directxmath/meson.build

```diff
@@ -0,0 +1,7 @@
+project('directxmath', 'cpp', version: '3.1.8')
```

Comment:
Where is this version `3.1.8` coming from? The release tag says `3.18`: https://github.com/microsoft/DirectXMath/releases/tag/dec2022

Reply:
uh, it seems to be inconsistent with version specified in cmake file then https://github.com/microsoft/DirectXMath/blob/dec2022/CMakeLists.txt#L6

Reply:
I guess it might be mistake in their cmake file but I'd follow the cmake project version

Reply:
Filed an issue https://github.com/microsoft/DirectXMath/issues/169